proto/opamp.proto

```diff
@@ -394,6 +419,31 @@ message OtherConnectionSettings {
     // Other connection settings. These are Agent-specific and are up to the Agent
     // interpret.
     map<string, string> other_settings = 4;
+
+    // Optional connection specific TLS settings.
+    // Status: [Development]
+    TLSConnectionSettings tls = 5;
+}
+
+// Status: [Development]
+message TLSConnectionSettings {
+  // Provides CA cert contents as a string.
+  string ca_pem_contents = 1;
+
+  // Load system CA pool alongside any passed CAs.
+  bool include_system_ca_certs_pool = 2;
+
+  // skip certificate verification.
+  bool insecure_skip_verify = 3;
```

Comment:
Isn't this a must if the server sends a new CA certificate? The agent has no way to verify it anway since the CA is unknown to the agent.

Reply:
Again, this setting is meant to be more generic and not strictly tied to the CA distribution flow.